learningresources/tests/test_api.py

```diff
@@ -61,6 +64,10 @@ def test_create(self):
             create_course(**self.kwargs)
         self.assertTrue(course_count() == after)
 
+        # NOT NULL constraint fails on org
+        with self.assertRaises(IntegrityError):
```

Comment:
Unfortunately the exception string changes when running against SQLite and against postgres